src/cmd/internal.go

```diff
@@ -111,9 +111,9 @@ var generateCLIDocs = &cobra.Command{
 	},
 }
 
-var configSchemaCmd = &cobra.Command{
-	Use:     "config-schema",
-	Aliases: []string{"c"},
+var genConfigSchemaCmd = &cobra.Command{
+	Use:     "generate-config-schema",
```

Comment:
```suggestion
	Use:     "gen-config-schema",
```

To better match `zarf tools gen-pki/gen-key`

Reply:
Same would apply to `generate-cli-docs`